docs/types/at-a-glance.md

```diff
@@ -31,7 +31,7 @@ The Pony type system offers a lot of guarantees, even more than other statically
 * Your program will never deadlock.
 * Your code will always be capabilities-secure.
 * All message passing is
-[causal](https://courses.cs.vt.edu/~cs5204/fall00/causal.html). (Not ca**su**al!)
+[causal](https://courses.cs.vt.edu/~cs5204/fall00/causal.html). (Not `ca**su**al`!)
```

Comment:
The backticks will show this word verbatim, perhaps it'd be better to set the entire word in bold?

Reply:
That seems reasonable to me

Reply:
I pushed that change.